### ISSUE TITLE
pill-ify /me?

### ISSUE THREAD

**Issue by turt2live** [vector-im/riot-web#4941]
https://matrix.to/#/!DgvjtOljKujDBrxyHk:matrix.org/$15041143541535444qTDUu:matrix.org
> Feature request: the me in a /me is not a pill.

**Comment by lampholder** [comment]
This would be nice to have.

**Comment by t3chguy** [comment]
@lampholder blocked on design
see conversation at:
https://matrix.to/#/!DdJkzRliezrwpNebLk:matrix.org/$1505156139512eoNeg:riot.ovh

**Comment by lampholder** [comment]
Oh my!

Yes, I can see the problem. What are your thoughts on this maybe being more hassle than it's worth?

**Comment by turt2live** [comment]
> What are your thoughts on this maybe being more hassle than it's worth?

Sorry for missing your question :(

I think it's too much hassle to be honest. It sounds great in theory, but in practice it's a nightmare. I'd recommend that if others feel strongly for this feature that they open a new issue.
